wordpress-mobile/WordPress-iOS
# Fix alternate app icons

Fixes an issue causing the app to be rejected by Apple. Also fixes an issue causing custom icons not to work on iPad.

**To test:**
- Run on iPhone simulator and ensure all alternate icons work
- Run on iPad Pro 12.9" simulator and ensure all alternate icons work
- In App Store Connect, note that `12.8.0.1.1` passes validation

Update release notes:
